.github/workflows/publish-release-npm.yml

```diff
@@ -54,7 +54,7 @@ jobs:
               uses: lauchacarro/Azure-Storage-Action@v1.1
               with:
                   # Connection String of the Azure Storage Container
-                  connection-string: ${{ secrets.AZURE_CDN_SECRET }}
+                  connection-string: ${{ secrets.AZURE_CDN_STUDIO_PRD_CONNECTION_STRING }}
                   # Folder what contains the files for upload to Storage Container
                   folder: upload
```

Comment:
Not sure what this means for the eventual CDN path of the SDK. It should be `<cdn-domain>/sdk/<version>/abc.js` by the looks of it, with current configuration `/builds/` will be part of the path?

Reply:
Also, `blob-container-name` should it be `sdk`? It refers to the container on the root level of the Azure Storage account IIRC

Reply:
blob-container-name should be correct being SDK, storage account = stgrafxstudioprdpublic => container = SDK.
SA is defined by the connectionstring secret